.gitignore

```diff
@@ -12,6 +12,7 @@ coverage
 *.log
 yarn.lock
 /artifacts
+/flaky-tests.json
```

Comment:
If we only run `flaky-tests-reporter` in CI then we shouldn't need to ignore this file, yeah?